Detections/AWSCloudTrail/AWS_ConsoleLogonWithoutMFA.yaml

```diff
@@ -22,16 +23,12 @@ relevantTechniques:
   - T1078
 query: |
 
-  let timeframe = 1d;
+  let timeframe = 1h;
   AWSCloudTrail
-  | where TimeGenerated >= ago(timeframe)
+  | where TimeGenerated > ago(timeframe)
   | where EventName == "ConsoleLogin" 
-  | extend AdditionalData = todynamic(AdditionalEventData)
-  | extend MFAUsed = AdditionalData["MFAUsed"]
-  | extend RElements = todynamic(ResponseElements)
-  | extend CLogin = RElements["ConsoleLogin"]
-  | where MFAUsed != "Yes"
-  | where CLogin != "Failure"
-  | project TimeGenerated, EventName, EventTypeName, CLogin, MFAUsed, UserIdentityAccountId,  UserIdentityPrincipalid, UserAgent, 
-  UserIdentityUserName, SessionMfaAuthenticated, SourceIpAddress, AWSRegion, AdditionalData, RElements
+  | extend MFAUsed = tostring(parse_json(AdditionalEventData).MFAUsed), LoginResult = tostring(parse_json(ResponseElements).ConsoleLogin)
+  | where MFAUsed != "Yes" and LoginResult != "Failure"
+  | summarize by TimeGenerated, EventName, EventTypeName, LoginResult, MFAUsed, UserIdentityAccountId,  UserIdentityPrincipalid, UserAgent, 
```

Comment:
Can we aggregate on min() and max() for TimeGenerated and any others to reduce the resulting rowset?